packages/ppx_jsobject_conv/ppx_jsobject_conv.0.5.0/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+version: "0.5.0"
+maintainer: "Roma Sokolov <sokolov.r.v@gmail.com>"
+authors: [ "Roma Sokolov <sokolov.r.v@gmail.com>" ]
+license: "MIT"
+homepage: "https://github.com/little-arhat/ppx_jsobject_conv"
+bug-reports: "https://github.com/little-arhat/ppx_jsobject_conv/issues"
+dev-repo: "git://github.com/little-arhat/ppx_jsobject_conv.git"
+tags: [ "syntax" "jsoo" "javascript" ]
+build: [
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "js_of_ocaml" {>= "2.8"}
+  "ppx_type_conv" {>= "v0.10.0"}
+  "ppx_driver"
+  "ppx_core"
+  "ppx_metaquot"
+  "ocaml-migrate-parsetree" {>= "0.4"}
+  "jbuilder" {build & >= "1.0+beta12"}
+  "ocamlfind"    {build}
```

Comment:
You likely don't need findlib as a dependency. Jbuilder doesn't use findlib.

Reply:
True, missed that one, thanks!